src/Skia/Perspex.Skia.Android/AndroidRenderTarget.cs

```diff
@@ -14,6 +14,8 @@
 
 namespace Perspex.Skia
 {
+/* This lives in shared project now, but we may need an android specific implementation anyways
```

Comment:
No problem, I was not sure if this would you guys review the changes going on. I can clean out all dead code.